packages/monitors/src/OptimisticOracleContractMonitor.js

```diff
@@ -123,20 +138,30 @@ class OptimisticOracleContractMonitor {
 
     for (let event of latestEvents) {
       const mrkdwn =
-        createEtherscanLinkMarkdown(event.proposer, this.contractProps.networkId) +
+        createEtherscanLinkMarkdown(
+          this.oracleType === OptimisticOracleType.OptimisticOracle ? event.proposer : event.request.proposer,
+          this.contractProps.networkId
+        ) +
         ` proposed a price for the request made by ${event.requester} at the timestamp ${event.timestamp} for the identifier: ${event.identifier}. ` +
-        `The proposal price of ${this.formatDecimalString(event.proposedPrice)} will expire at ${
-          event.expirationTimestamp
+        `The proposal price of ${this.formatDecimalString(
+          this.oracleType === OptimisticOracleType.OptimisticOracle ? event.proposedPrice : event.request.proposedPrice
+        )} will expire at ${
+          this.oracleType === OptimisticOracleType.OptimisticOracle
+            ? event.expirationTimestamp
+            : event.request.expirationTime
         }. ` +
         `The ancillary data field is ${event.ancillaryData || "0x"}. ` +
-        `Collateral currency address is ${event.currency}. ` +
+        `Collateral currency address is ${
+          this.oracleType === OptimisticOracleType.OptimisticOracle ? event.currency : event.request.currency
+        }. ` +
         `tx: ${createEtherscanLinkMarkdown(event.transactionHash, this.contractProps.networkId)}`;
 
       // The default log level should be reduced to "info" for funding rate identifiers:
       this.logger[
         this.logOverrides.proposedPrice || (this._isFundingRateIdentifier(event.identifier) ? "info" : "error")
       ]({
         at: "OptimisticOracleContractMonitor",
+        type: this.oracleType,
```

Comment:
I dont think you should do it like this. rather, add the `type` into the markdown. The longer is not designed to deal with more than just a `at`, `message` `mrkdwn` and `notificationPath`, in the event the log produced contains `mrkdwn`.

Reply:
ok

Reply:
I'm going to add the `message` key for each info or higher log, ok?

Reply:
for example:

```
{
        at: "OptimisticOracleContractMonitor",
        message: `${this.oracleType}: Price Settlement Alert 🏧!`,
        mrkdwn,
        notificationPath: "optimistic-oracle",
}
```